config/initializers/cookies_serializer.rb

```diff
@@ -1,5 +0,0 @@
-# Be sure to restart your server when you modify this file.
-
-# Specify a serializer for the signed and encrypted cookie jars.
-# Valid options are :json, :marshal, and :hybrid.
-Rails.application.config.action_dispatch.cookies_serializer = :hybrid
```

Comment:
The default is now `:json`, which should be fine since we've been using the hybrid method for several months now (so things should have been updated to use the json format)